controllers/hibp.js

```diff
@@ -20,6 +22,27 @@ async function notify (req, res) {
 }
 
 
+async function breaches (req, res, next) {
+  const clientMostRecentBreachDateTime = new Date(req.headers["if-modified-since"]);
+  const serverMostRecentBreachDateTime = req.app.locals.mostRecentBreachDateTime;
+  if (clientMostRecentBreachDateTime < serverMostRecentBreachDateTime) {
+    res.append("Last-Modified", serverMostRecentBreachDateTime);
+    res.json(HIBP.filterOutUnsafeBreaches(req.app.locals.breaches));
```

Comment:
+1 to the comment above. If this /breaches endpoint is meant to be used w/ the add-on, should we be filtering out all the breaches? Or should we send the full list and let the client (add-on, in this case) decide what data it wants to use.